### ISSUE TITLE
Background glitches on Bubba 'N' Stix (Core Design).

### ISSUE THREAD

**Issue by Gianmarco72** [dirkwhoffmann/vAmiga#325]
There's a problem with the mountains in the background: they disappear with every Bubba's step.

Here they are:
![Mountain_OK](https://user-images.githubusercontent.com/45571833/80867502-b2bcfd80-8c94-11ea-90b6-f29a5dfd2dc2.jpg)
... now they are gone:
![Mountain_NO](https://user-images.githubusercontent.com/45571833/80867549-f3b51200-8c94-11ea-8074-61e4f54d254c.jpg)
Maybe a "sprite tricks"?

Also tested on UAE and it's ok 👍🏻

**Comment by dirkwhoffmann** [comment]
> Maybe a "sprite tricks"?

Excellent opportunity to try out the new stencils feature from the Monitor panel 😎.

<img width="957" alt="Bildschirmfoto 2020-05-02 um 17 31 36" src="https://user-images.githubusercontent.com/12561945/80868493-42fe4100-8c9b-11ea-86fb-5208b608d487.png">

Using the stencil tool, it is possible to cut out certain graphics layers such as the sprite layers. Applying the feature to this game results in the following image:

<img width="959" alt="Bildschirmfoto 2020-05-02 um 17 19 48" src="https://user-images.githubusercontent.com/12561945/80868518-835dbf00-8c9b-11ea-8a8d-76496c785d1a.png">

It is clearly not related to sprites this time.

Let's revert to the usual procedure: X-raying the patient 🤒.

<img width="1003" alt="Bildschirmfoto 2020-05-02 um 17 33 44" src="https://user-images.githubusercontent.com/12561945/80868547-aa1bf580-8c9b-11ea-9e06-cb54efce0fed.png">

Wooaaaaahhhh, another game that maxes out the Blitter to the last free DMA cycle 😬.

There is heavy Copper activity in the mountain area, so I bet it's another issue with register access timing. 

Next step: Put on safety goggles and have a look at the Copper list ☢️. We need to find out the registers that get modified by the Copper in this are.

**Comment by Gianmarco72** [comment]
> Excellent opportunity to try out the new stencils feature from the Monitor panel 😎.

Very cool!! 😃
I don't see the stencils feature ...
![stencils](https://user-images.githubusercontent.com/45571833/80869206-5ad7c400-8c9f-11ea-8e08-cdad888b6b00.jpg)
... will it be in a future release?

**Comment by dirkwhoffmann** [comment]
> I don't see the stencils feature ...

Latest hot stuff from vAmiga engineering 😎. Top secret. Not yet released.

It's a nice feature to quickly see where the visible pixels come from. E.g., here is the original Cabal intro which uses a two-layer scrolling effect:

<img width="1003" alt="Bildschirmfoto 2020-05-02 um 19 00 21" src="https://user-images.githubusercontent.com/12561945/80870587-b1490080-8ca7-11ea-8ecd-1b539a396295.png">

These pixels stem from playfield 1:

<img width="1003" alt="Bildschirmfoto 2020-05-02 um 19 00 42" src="https://user-images.githubusercontent.com/12561945/80870592-b9a13b80-8ca7-11ea-8fc1-5ae6a392c3cb.png">

And those from playfield 2:

<img width="1003" alt="Bildschirmfoto 2020-05-02 um 19 00 33" src="https://user-images.githubusercontent.com/12561945/80870594-bdcd5900-8ca7-11ea-8d27-ad678c797e59.png">

**Comment by dirkwhoffmann** [comment]
> It is clearly not related to sprites this time.

Well, it turned out that this is not really true. Actually, it *is* all about sprites. The code changes that fixed "Brian the Lion" fixed this issue, too.

The following image shows all sprite layers together:

<img width="959" alt="Bildschirmfoto 2020-05-04 um 08 30 43" src="https://user-images.githubusercontent.com/12561945/80942676-aa280c80-8de5-11ea-9fd3-d076ab7e5490.png">

And here is sprite 7 only:

<img width="1003" alt="Bildschirmfoto 2020-05-04 um 08 31 35" src="https://user-images.githubusercontent.com/12561945/80942689-b1e7b100-8de5-11ea-9e75-45745f76b2a6.png">

Bottom line: The mountain parallax effect is all done with sprites.

There is another parallax layer at the bottom which has been done the traditional way (in dual-playfield mode). These pixels belong to the second playfield: 

<img width="1003" alt="Bildschirmfoto 2020-05-04 um 08 29 55" src="https://user-images.githubusercontent.com/12561945/80943045-90d39000-8de6-11ea-8e52-ffdc165753ff.png">
